nemo/collections/asr/parts/utils/transcribe_utils.py

```diff
@@ -289,7 +289,7 @@ def write_transcription(
     else:
         pred_text_attr_name = 'pred_text'
 
-    with open(cfg.output_filename, 'w', encoding='utf-8') as f:
+    with open(cfg.output_filename, 'w', encoding='utf-8', newline='\n') as f:
```

Comment:
Hmm is this newline part needed?

Reply:
So that it works correctly on Windows, which is my primary Nemo working environment. newline='\n' sets the file to use UNIX EOL, so this change is harmless on Unix, but on Windows is necessary to output UNIX EOL files, which makes it in parity with Nemo on unix